pkg/integration/hosts.go

```diff
@@ -15,7 +15,10 @@
 package integration
 
 import (
+	apis "github.com/metalkube/baremetal-operator/pkg/apis/metalkube/v1alpha1"
```

Comment:
is this arbitrary?

Reply:
I'm not sure what you mean.

Reply:
Just wondering if it follows a convention.

Reply:
It most likely does. But this is a link to the actual package. We're only consuming it.